libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -704,6 +704,14 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("GND_EFF_DZ", 7, NavEKF3, _baroGndEffectDeadZone, 4.0f),
 
+    // @Param: PRIMARY
+    // @DisplayName: Primary core number
+    // @Description: The code number (IMU index) that will be used as the primary EKF core on startup. While disarmed the EKF will force the use of this core. A value of 0 corresponds to the first IMU.
```

Comment:
I guess this number really is the EKF core number rather than the IMU number because some IMUs could be unused because of the EK3_IMU_MASK parameter value.  I think we should change this param's description to consistently use the word "core" instead of IMU.

The EKF co**r**e that will be used as the primary core on startup. While disarmed the EKF will force the use of this core. A value of 0 corresponds to the first core.

Reply:
good point, fixed